config/ember-try.js

```diff
@@ -1,25 +1,35 @@
-/*jshint node:true*/
+/* eslint-env node */
 module.exports = {
   scenarios: [
     {
-      name: 'ember-1.13',
+      name: 'ember-lts-2.4',
       bower: {
         dependencies: {
-          'ember': '~1.13.0'
+          'ember': 'components/ember#lts-2-4'
         },
         resolutions: {
-          'ember': '~1.13.0'
+          'ember': 'lts-2-4'
+        }
+      },
+      npm: {
+        devDependencies: {
+          'ember-source': null
         }
       }
     },
     {
-      name: 'ember-lts-2.4',
+      name: 'ember-lts-2.8',
       bower: {
         dependencies: {
-          'ember': 'components/ember#lts-2-4'
+          'ember': 'components/ember#lts-2-8'
```

Comment:
since we want this to supprt 2.12, do you mind adding 2.12 here? (maybe remove 2.4, and then keep 2.8 and 2.12)

Reply:
2.12 is actually tested through the `ember-default` scenario, which tests whichever version is specified in the `package.json` file.